core/src/main/java/io/seata/core/store/db/LogStoreDataBaseDAO.java

```diff
@@ -45,6 +48,17 @@
 @LoadLevel(name = "db")
 public class LogStoreDataBaseDAO implements LogStore, Initialize {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(LogStoreDataBaseDAO.class);
+
+    /**
+     * The transaction name key
+     */
+    private static final String TRANSACTION_NAME_KEY = "TRANSACTION_NAME";
+    /**
+     * The transaction name default size is 64
+     */
+    private static final int TRANSACTION_NAME_DEFAULT_SIZE = 64;
```

Comment:
128, consistent with the size of the transaction_name in the current table structure.

Reply:
fix done